### DISCUSSION TITLE
Add additional query options to the "Create a useful People search" scenario

### DISCUSSION THREAD

**Discussion by kasperbolarsen** [microsoft-search/pnp-modern-search#2686] [Ideas]
The idea is to add some additional options to Davids scenario.

I often use these to filter the User Profiles:

SPS-HideFromAddressLists  ( from Exchange, used to exclude users , often former employees)
-preferredname:admin*
-preferredname:test*
owstaxIdSPShDepartment ( only show user profiles with a value in the Department field)
JobTitle  ( same as for owstaxIdSPShDepartment  )

(MobilPhone:0* OR MobilPhone:1* OR MobilPhone:2* OR MobilPhone:3* OR MobilPhone:4* OR MobilPhone:5* OR MobilPhone:6* OR MobilPhone:7* OR MobilPhone:8* OR MobilPhone:9*)   Ugly as sin, but it works

**Comment by wobba** [comment]
I think this makes sense and would be useful. Right now it's also tied to the search provider. But we could maybe set a default query when you pick e.g. the People template or result source?

**Reply by kasperbolarsen** [reply]
I guess each set of User Profiles often reflects the messy on-premises AD and the set of properties required to get a useful dataset therefore must be tailored to each tenant?
But once I have figured out how to build the documentation I'll put the options above in a PR :-)

**Reply by wobba** [reply]
I have some useful exclusions in https://www.techmikael.com/2018/01/excluding-sharepoint-online-service.html as well

**Comment by kasperbolarsen** [comment]
Covered by https://microsoft-search.github.io/pnp-modern-search/scenarios/Create-a-useful-People-Search/
